src/w3c/templates/sotd.js

```diff
@@ -204,26 +204,19 @@ export default (conf, opts) => {
                         : ""
                     }
                     <p data-deliverer="${conf.isNote ? conf.wgId : null}">
-                      ${
-                        !conf.isIGNote
-                          ? html`
-                              This document was produced by
-                              ${conf.multipleWGs ? "groups" : "a group"}
-                              operating under the
-                              <a
-                                href="https://www.w3.org/Consortium/Patent-Policy/"
-                                >W3C Patent Policy</a
-                              >.
-                            `
-                          : ""
-                      }
+                      This document was produced by
```

Comment:
That sentence isn't needed for IG Notes.

Reply:
hopefully good now.